### ISSUE TITLE
Data usage issues during self-supervised pre-training

### ISSUE THREAD

**Issue by JAAY-AI** [AIRMEC/im4MEC#5]
I would like to ask, after self-supervised pre-training on a certain data set, is it not possible to perform downstream classifier training on this data set (because the data distribution during pre-training is exposed to the classifier), so I Confused about your practice of pre-training on multiple centers and then training the classifier on those centers.

**Comment by jjhbw** [comment]
Yes, technically, the pre-training exposes the domain to the entire pipeline if you do that. Pre-training should take place after data split. You can simply do the pre-training on the training set only.

Obviously, you should always aim to validate the final model using an independent test set that is not used for any of the training steps (self-supervised or the task-level classifier). Ideally a training set derived from an external cohort (unseen domain), as we did in the im4MEC paper.

**Comment by JAAY-AI** [comment]
@jjhbw Thank you very much for your work and answers!
